### DISCUSSION TITLE
Are there any plans to upgrade to .NET 7.0?

### DISCUSSION THREAD

**Discussion by copecone** [ppy/osu-framework#6029] [Q&A] [answered]
The current .NET 6.0 is very good, but I heard that there were some .NET performance improvements in 7.0. Do you have any future plans to upgrade to .NET 7.0?

**Comment by Joehuu** [comment, marked as answer]
There are other priorities right now (ranked play). That aside, .NET 7.0 [ends support](https://github.com/dotnet/core/blob/main/release-notes/7.0/supported-os.md#support-changes-from-net-60) of Windows 7 and 8.1. osu!(lazer) aims to support old OSes if possible (it would support Windows 7 if realm did so).

I believe there needs to be a change in the tooling (which will require a bit of time) to build .NET 6 and 7 builds if we still want to support those OSes.

**Reply by copecone** [reply]
Oh, and considering that .NET 7.0 has ended support for Windows 7 and 8.1, not upgrading to .NET 7.0 was something I hadn't considered. However, support for Windows 7 and Windows 8.1 may *still* be important, so I like this decision!

(I used Google Translator, so the sentences may sound strange)
